katcp/testutils.py

```diff
@@ -1045,6 +1045,10 @@ def setup_sensors(self):
             Sensor.FLOAT, "a.float", "A Float.", "",
             [-123.4, 123.4],
             timestamp=12345, status=Sensor.NOMINAL, value=12))
+        self.add_sensor(DeviceTestSensor(
```

Comment:
My OCD got triggered :rofl: 

```suggestion
        self.add_sensor(
            DeviceTestSensor(
                Sensor.FLOAT, 
                "a.floatwithzero", 
                "A Float with zero.", 
                "",
                [-123.0, 123.0],
                timestamp=12345,
                status=Sensor.NOMINAL,
                value=12))
```

Reply:
@mmphego Do you want me to run black on all the code?

Reply:
Would be nice to run black before committing IMHO.

Reply:
thanks but from my experience black conflicts with our linter. see http://ci.camlab.kat.ac.za/blue/organizations/jenkins/katcp-multibranch/detail/user%2Ftockards%2Fblack-test-server/1/pipeline

Unless there are settings and things I need to suppress in black.

Reply:
You could run it as: `black -t py27 -t py36 -l 90 <file>`